app/adapters/application.js

```diff
@@ -3,17 +3,16 @@ import { Adapter } from 'ember-pouch';
 import PouchAdapterUtils from "hospitalrun/mixins/pouch-adapter-utils";
 
 export default Adapter.extend(PouchAdapterUtils, {
-    pouchDBService: Ember.inject.service('pouchdb'),
-    
-    mainDB: Ember.computed.alias('pouchDBService.mainDB'),
-    db:  Ember.computed.alias('pouchDBService.mainDB'),
-    
+    database: Ember.inject.service(),
+    mainDB: Ember.computed.alias('database.mainDB'),
+    db:  Ember.computed.alias('database.mainDB'),
```

Comment:
can we merge mainDB and db into one since they point to the same thing. on that note we could just use database or call database db?